sample-service/src/main/java/uk/ac/ox/ndph/mts/sample_service/security/AADWebSecurityConfig.java

```diff
@@ -0,0 +1,49 @@
+package uk.ac.ox.ndph.mts.sample_service.security;
+
+import com.microsoft.azure.spring.autoconfigure.aad.AADAppRoleStatelessAuthenticationFilter;
```

Comment:
Is there any reason to use the Microsoft AD library here rather than the spring standard - ties us in?  I think we decided to stick to the JWT/OpenID standards here not use any extended AD claims in the token, so org.springframework.security.oauth2.jwt should be sufficient?

Reply:
By project policy, we can't use an Azure-specific library in a core component - it seems like the Spring OAuth library should work here. I know Kate has done quite a bit of work with it, so she may be able to assist. The only way it would be acceptable is if this was to unblock some other work, and if someone was assigned to replacing the Azure library with the Spring library as a priority.

Reply:
Created a story to try and change that to use the Spring Security libraries: https://ndph-arts.atlassian.net/browse/ARTS-591